5-Access-Authentication-and-Authorization/5.5-User-Accounts-and-Environment.bats

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bats
+
+load 5.5.4-helper
+load 5.5.5-helper
+
+@test "5.5.1.1 Ensure minimum days between password changes is configured (Automated)" {
+    run bash -c "grep PASS_MIN_DAYS /etc/login.defs | grep --invert-match \"\#\""
+    [ "$status" -eq 0 ]
+    [[ "$output" != "" ]]
+    MINDAYS=(${output//PASS_MIN_DAYS/ }) # get the number from the string
+    [[ "$MINDAYS" -gt 0 ]]
+    run bash -c "grep -E ^[^:]+:[^\!*] /etc/shadow | cut -d: -f1,4"
+    [ "$status" -eq 0 ]
+    while IFS=: read -r line; do
+        MINDAYS=(${line//:/ }) # get the number from the string
+        [[ "${MINDAYS[1]}" -gt 0 ]]
+    done <<< "$output"
+}
+
+@test "5.5.1.2 Ensure password expiration is 365 days or less (Automated)" {
+    run bash -c "grep PASS_MAX_DAYS /etc/login.defs | grep --invert-match \"\#\""
+    [ "$status" -eq 0 ]
+    [[ "$output" != "" ]]
+    MAXDAYS=(${output//PASS_MAX_DAYS/ }) # get the number from the string
+    [[ "$MAXDAYS" -lt 366 ]]
+    run bash -c "grep -E '^[^:]+:[^!*]' /etc/shadow | cut -d: -f1,5"
+    [ "$status" -eq 0 ]
+    while IFS=: read -r line; do
+        MAXDAYS=(${line//:/ }) # get the number from the string
+        [[ "${MAXDAYS[1]}" -lt 366 ]]
+    done <<< "$output"
+}
+
+@test "5.5.1.3 Ensure password expiration warning days is 7 or more (Automated)" {
+    run bash -c "grep PASS_WARN_AGE /etc/login.defs | grep --invert-match \"\#\""
+    [ "$status" -eq 0 ]
+    [[ "$output" != "" ]]
+    WARNAGE=(${output//PASS_WARN_AGE/ }) # get the number from the string
+    [[ "$WARNAGE" -gt 6 ]]
+    run bash -c "grep -E ^[^:]+:[^\!*] /etc/shadow | cut -d: -f1,6"
+    [ "$status" -eq 0 ]
+    while IFS=: read -r line; do
+        WARNAGE=(${line//:/ }) # get the number from the string
+        [[ "${WARNAGE[1]}" -gt 6 ]]
+    done <<< "$output"
+}
+
+@test "5.5.1.4 Ensure inactive password lock is 30 days or less (Automated)" {
+    run bash -c "useradd -D | grep INACTIVE"
+    [ "$status" -eq 0 ]
+    [[ "$output" != "" ]]
+    INACTIVE=(${output//INACTIVE=/ }) # get the number from the string
+    [[ "$INACTIVE" != -1 ]]
+    [[ "$INACTIVE" -lt 31 ]]
+    run bash -c "grep -E ^[^:]+:[^\!*] /etc/shadow | cut -d: -f1,7"
```

Comment:
This command is not according to the CIS document (`awk -F: '(/^[^:]+:[^!*]/ && ($7~/(\s*|-1)/ || $7>30)){print $1 " " $7}'
/etc/shadow`)